html/browsers/the-window-object/window-indexed-properties-strict.html

```diff
@@ -24,6 +24,8 @@
     window[0] = "foo";
   });
   assert_throws(new TypeError(), () => Object.defineProperty(window, 0, { value: "bar" }))
+  assert_throws(new TypeError(), () => Object.defineProperty(window, 0, { get() { return "baz" } }))
+  assert_throws(new TypeError(), () => Object.defineProperty(window, 1, { set() { return "qux" } }))
   assert_equals(window[0],
                 document.getElementsByTagName("iframe")[0].contentWindow);
```

Comment:
Should we be asserting window[1] is undefined, like the next test does? I'm kind of confused about the test split honestly.